example/print-manager-v2-example.ts

```diff
@@ -80,53 +82,52 @@ const onScaleChange = event => {
  *
  */
 const registerPrintHandler = () => {
-  const printBtn = document.querySelector('button#print');
+  const printBtn: any = document.querySelector('button#print');
   printBtn.onclick = onPrintClick;
 };
 
 /**
  *
  */
 const onPrintClick = () => {
-  printProvider.print(true)
-    .catch(error => {
-      console.log('Error while printing: ' + error);
-    });
+  printProvider?.print(true)?.catch(error => {
+    console.log('Error while printing: ' + error);
+  });
 };
 
 /**
  *
  */
 const fillCombos = () => {
-  const layoutSelect = document.querySelector('select#layout-select');
-  printProvider.getLayouts().forEach(layout => {
+  const layoutSelect: any = document.querySelector('select#layout-select');
+  printProvider.getLayouts().forEach((layout: any) => {
     const option = document.createElement('option');
     option.text = layout.name;
-    layoutSelect.add(option);
+    layoutSelect?.add(option);
   });
   layoutSelect.value = printProvider.getLayout().name;
   layoutSelect.onchange = onLayoutChange;
 
-  const dpiSelect = document.querySelector('select#dpi-select');
-  printProvider.getDpis().forEach(dpi => {
+  const dpiSelect: any = document.querySelector('select#dpi-select');
+  printProvider.getDpis().forEach((dpi: any) => {
     const option = document.createElement('option');
     option.text = dpi.name;
-    dpiSelect.add(option);
+    dpiSelect?.add(option);
   });
   dpiSelect.value = printProvider.getDpi().name;
   dpiSelect.onchange = onDpiChange;
 
-  const formatSelect = document.querySelector('select#format-select');
-  printProvider.getOutputFormats().forEach(format => {
+  const formatSelect: any = document.querySelector('select#format-select');
+  printProvider.getOutputFormats().forEach((format: any) => {
     const option = document.createElement('option');
     option.text = format.name;
     formatSelect.add(option);
   });
   formatSelect.value = printProvider.getOutputFormat().name;
   formatSelect.onchange = onFormatChange;
 
-  const scaleSelect = document.querySelector('select#scale-select');
-  printProvider.getScales().forEach(scale => {
+  const scaleSelect: any = document.querySelector('select#scale-select');
+  printProvider.getScales().forEach((scale: any) => {
     const option = document.createElement('option');
     option.text = scale.name;
     scaleSelect.add(option);
```

Comment:
The example files will be loaded as plain js files in the examples sites, e.g. [here](https://github.com/terrestris/mapfish-print-manager/blob/master/example/print-manager-v2-example.html#L28). So I think we should revert this.

Reply:
True, fixed in 1fe2c12